api/logic/load_balancer_logic.go

```diff
@@ -123,70 +123,51 @@ func (this *L0LoadBalancerLogic) UpdateLoadBalancer(loadBalancerID string, ports
 }
 
 func (this *L0LoadBalancerLogic) doesLoadBalancerTagExist(environmentID, name string) (bool, error) {
-	filter := map[string]string{
-		"type":           "load_balancer",
-		"environment_id": environmentID,
-		"name":           name,
-	}
-
-	tags, err := this.TagData.GetTags(filter)
+	tags, err := this.TagStore.SelectByQuery("load_balancer", "")
 	if err != nil {
 		return false, err
 	}
 
-	return len(tags) > 0, nil
+	ewts := tags.GroupByEntity().
```

Comment:
Nice, this is a lot more readable. What is the meaning of the `ewts` name, though?